components/script/dom/identityhub.rs

```diff
@@ -132,6 +138,20 @@ impl Identities {
         }
     }
 
+    pub fn create_bind_group_id(&mut self, backend: Backend) -> BindGroupId {
```

Comment:
Why aren't we doing this simpler, like in https://searchfox.org/mozilla-central/rev/ad6234148892bc29bf08d736604ac71925138040/gfx/wgpu/wgpu-remote/src/lib.rs#277

Reply:
Yes, you're right! This could be simpler. I will remove the code duplication in another pull request.

Reply:
I've just created a PR for this: #25648